.env.development.sample

```diff
@@ -1,11 +1,12 @@
 # Define ENV variables for development environment
-DATABASE_URL="postgresql://localhost/ruby_jobs_development"
+HANAMI_ENV=development
+DATABASE_URL=postgres://safelylaunch@postgres/core
 SERVE_STATIC_ASSETS="true"
```

Comment:
we can remove double-quotes everywhere in this file

Reply:
Sure, thx